specification/provisioningservices/resource-manager/readme.md

```diff
@@ -26,16 +26,16 @@ These are the global settings for the API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2017-08
```

Comment:
Is removing the previous tag intentional?

Reply:
unintentional will fix this